src/utils/formatting.ts

```diff
@@ -28,12 +28,14 @@ export const ruleToString = (seconds: number): string => {
       return acc
     }
 
-    return `${acc} ${v} ${k}`
+    // removes the trailing plural 's' if magnitude is singular
+    const k_singular_or_plural = v <= 1 ? k.slice(0, -1) : k
```

Comment:
I think you want to camelCase this variable, right?

Reply:
Done!